amgut/handlers/add_sample.py

```diff
@@ -20,12 +21,24 @@ class AddSample(BaseHandler):
     @authenticated
     def post(self):
         # Required vars
-        barcode = self.get_argument('barcode')
-        sample_date = self.get_argument('sample_date')
-        sample_time = self.get_argument('sample_time')
-        notes = self.get_argument('notes')
-        participant_name = self.get_argument('participant_name')
-        sample_site = self.get_argument('sample_site', '')
+        participant_name = self.get_argument('participant_name',
+                                             'environmental')
+        form = self.build_form()
```

Comment:
how do these changes fit into this PR and what is the relevance of these changes?